src/Inkluzitron/Modules/BdsmTestOrg/BdsmModule.cs

```diff
@@ -31,15 +34,25 @@ public class BdsmModule : ModuleBase
             RegexOptions.Multiline
         );
 
+        private GraphPainter GraphPainter { get; }
+        private DiscordSocketClient Client { get; }
         private BotDatabaseContext DbContext { get; }
         private ReactionSettings ReactionSettings { get; }
         private BdsmTestOrgSettings Settings { get; }
 
-        public BdsmModule(BotDatabaseContext dbContext, ReactionSettings reactionSettings, BdsmTestOrgSettings bdsmTestOrgSettings)
+        public BdsmModule(BotDatabaseContext dbContext, ReactionSettings reactionSettings, BdsmTestOrgSettings bdsmTestOrgSettings, DiscordSocketClient client, OpenSansFontManager fontMgr)
         {
+            GraphPainter = new GraphPainter
```

Comment:
Trochu se mi nelíbí, že se to tu vytváří v konstruktoru a že to není v té service. Co když to někdo bude chtít použít jinde? To nastavení fontů bych přesunul až do service a udělal z toho service. Ať to proleze přes kontejner.